packages/analytics-js-integrations/src/integrations/RedditPixel/browser.js

```diff
@@ -29,32 +35,25 @@ class RedditPixel {
   }
 
   init() {
-    logger.debug('===In init RedditPixel===');
-
     loadNativeSdk(this.advertiserId);
   }
 
   isLoaded() {
-    logger.debug('===In isLoaded RedditPixel===');
     return !!(window.rdt && window.rdt.advertiserId === this.advertiserId);
   }
 
   isReady() {
-    logger.debug('===In isReady RedditPixel===');
-    return !!(window.rdt && window.rdt.advertiserId === this.advertiserId);
+    return this.isLoaded();
   }
 
   identify(rudderElement) {
-    logger.debug('===In RedditPixel identify===');
     window.rdt('track', 'SignUp');
   }
 
   track(rudderElement) {
-    logger.debug('===In RedditPixel track===');
-
     const { event } = rudderElement.message;
     if (!event) {
-      logger.error('Event name is not present');
+      logger.error(` ${DISPLAY_NAME} : Event name is not present`);
```

Comment:
Isn't DISPLAY_NAME added to the logger instance again?

Reply:
updated